sky/core.py

```diff
@@ -205,6 +205,8 @@ def start(
         sky.exceptions.NotSupportedError: if the cluster to restart was
           launched using a non-default backend that does not support this
           operation.
+        sky.exceptions.ClusterOwnerIdentitiesMismatchError: if the cluster to
+            restart was launched by a different user.
```

Comment:
Is this from `backend_utils.refresh_cluster_status_handle()`? There are other places in this module that call this.

Reply:
It is from the `refresh_cluster_status_handle` (which will only be called when the cluster has autostop set up or is a spot cluster) or the `backend.provision` (we check the identity before provisioning the cluster.)

Reply:
I meant that: should we update the `Raises:` section for all code paths that may raise? 
- `autostop()`
- `queue/cancel/tail_logs/download_logs/job_status` -> `_check_cluster_available` -> `backend_utils.refresh_cluster_status_handle`
- `spot_*()` -> `_is_spot_controller_up` -> `backend_utils.refresh_cluster_status_handle`

Reply:
That is a great point! In the latest commit, I catch the error when the `refresh_cluster_status_handle` is called for those operations so that they will not be blocked by the identity check.